### PR TITLE
Remove obsolete drawings of diagram layer

### PR DESCRIPTION
**Pull request by christiankral**
Incorporating #3056

In order to review this PR it makes sense to have a look at the diagram layers of models in the following packages:
- Modelica.Blocks.Continuous
- Modelica.Blocks.Discrete
- Modelica.Blocks.Continuous
- Modelica.Blocks.Discrete
- Modelica.Blocks.Interfaces
- Modelica.Blocks.Logical
- Modelica.Blocks.Math
- Modelica.Blocks.Math
- Modelica.Blocks.Nonlinear
- Modelica.Blocks.Routing
- Modelica.Blocks.Sources
- Modelica.Blocks.Tables
- Modelica.ComplexBlocks
- Modelica.Magnetic.FluxTubes.Basic
- Modelica.Magnetic.FluxTubes.Sources
- Modelica.Magnetic.FundamentalWave.Components
- Modelica.Magnetic.QuasiStatic.FundamentalWave.Components
- Modelica.Thermal.HeatTransfer.Components
- Modelica.Thermal.HeatTransfer.Sources
- Modelica.Thermal.HeatTransfer.Components
- Modelica.Thermal.HeatTransfer.Celsius
- Modelica.Thermal.HeatTransfer.Fahrenheit
- Modelica.Thermal.HeatTransfer.Rankine
- Modelica.Mechanics.Rotational.Components
- Modelica.Mechanics.Rotational.Sources
- Modelica.Mechanics.Translational.Components
- Modelica.Thermal.FluidHeatFlow.BaseClasses
- Modelica.Math
- Modelica.Electrical.Analog.Lines
- Modelica.Electrical.Analog.Semiconductors.Thyristor
- Modelica.Electrical.Analog.Sources

### CODE REVIEW

**Comment by christiankral** [comment]
I have a couple more packages to look at before opening this PR

**Comment by christiankral** [comment]
OK now, I guess. 

- Most redundant diagram drawings are now removed
- Wherever it seemed necessary to me, a made a copy/replacement of the diagram drawing and included it in the info layer

**Comment by dietmarw** [comment]
Just a general comment. After this clean up we have quite a selection of empty default diagram annotations that should be removed (I can do this with https://github.com/dietmarw/trimtrailingwhitespaces) but probably should be done later after this has been merged as I guess there are quite a couple of other models that reintroduced those annotations during the development cycle.

File: Modelica/ComplexBlocks.mo [hunk 1]
```diff
@@ -1327,10 +1327,9 @@ An error occurs if the elements of the input <code>u</code> is zero.
     block RealToComplex "Converts Cartesian representation to complex"
       extends Modelica.ComplexBlocks.Interfaces.ComplexSO;
       Blocks.Interfaces.RealInput re annotation (Placement(transformation(
-              extent={{-130,40},{-90,80}}), iconTransformation(extent={{-140,40},
-                {-100,80}})));
+              extent={{-140,40},{-100,80}}),iconTransformation(extent={{-140,40},{-100,80}})));
```
Comment (HansOlsson) [inline Modelica/ComplexBlocks.mo]: Why are these connectors moved?

File: Modelica/ComplexBlocks.mo [hunk 2]
```diff
@@ -1327,10 +1327,9 @@ An error occurs if the elements of the input <code>u</code> is zero.
...
       Blocks.Interfaces.RealInput im annotation (Placement(transformation(
-              extent={{-130,-80},{-90,-40}}), iconTransformation(extent={{-140,
+              extent={{-140,-80},{-100,-40}}),iconTransformation(extent={{-140,
```
Comment (HansOlsson) [inline Modelica/ComplexBlocks.mo]: Same question: Why are these connectors moved?

File: Modelica/ComplexBlocks.mo [hunk 3]
```diff
@@ -1356,10 +1355,10 @@ An error occurs if the elements of the input <code>u</code> is zero.
     block PolarToComplex "Converts polar representation to complex"
       extends Modelica.ComplexBlocks.Interfaces.ComplexSO;
       Blocks.Interfaces.RealInput len annotation (Placement(transformation(
-              extent={{-130,40},{-90,80}}), iconTransformation(extent={{-140,40},
+              extent={{-140,40},{-100,80}}),iconTransformation(extent={{-140,40},
```
Comment (HansOlsson) [inline Modelica/ComplexBlocks.mo]: Same question: Why are these connectors moved?

File: Modelica/ComplexBlocks.mo [hunk 4]
```diff
@@ -1356,10 +1355,10 @@ An error occurs if the elements of the input <code>u</code> is zero.
...
                 {-100,80}})));
       Blocks.Interfaces.RealInput phi(unit="rad") annotation (Placement(transformation(
-              extent={{-130,-80},{-90,-40}}), iconTransformation(extent={{-140,
+              extent={{-140,-80},{-100,-40}}),iconTransformation(extent={{-140,
```
Comment (HansOlsson) [inline Modelica/ComplexBlocks.mo]: Same question: Why are these connectors moved?

File: Modelica/Electrical/Analog/Interfaces.mo [hunk 5]
```diff
@@ -125,43 +125,17 @@ The reason could be that
 
   partial model TwoPin "Component with two electrical pins"
     SI.Voltage v "Voltage drop of the two pins (= p.v - n.v)";
-    PositivePin p "Positive electrical pin" annotation (Placement(
-          transformation(extent={{-110,-10},{-90,10}})));
-    NegativePin n "Negative electrical pin" annotation (Placement(transformation(extent={{
-              90,-10},{110,10}})));
```
Comment (HansOlsson) [inline Modelica/Electrical/Analog/Interfaces.mo]: This must be an accident. Removing the pins shouldn't be done.

File: Modelica/Electrical/Analog/Interfaces.mo [hunk 6]
```diff
@@ -519,9 +468,6 @@ The device temperature <strong>internalHeatPort.T</strong> can be used to descri
 
     parameter SI.Voltage offset=0 "Voltage offset";
     parameter SI.Time startTime=0 "Time offset";
-    replaceable Modelica.Blocks.Interfaces.SignalSource signalSource(
-        final offset = offset, final startTime=startTime)
-    annotation (Placement(transformation(extent={{70,70},{90,90}})));
```
Comment (HansOlsson) [inline Modelica/Electrical/Analog/Interfaces.mo]: That change seems really good, but unrelated to the PR.
How did the signalSource appear?

File: Modelica/Electrical/Analog/Interfaces.mo [hunk 7]
```diff
@@ -544,9 +490,6 @@ The device temperature <strong>internalHeatPort.T</strong> can be used to descri
     extends OnePort;
     parameter SI.Current offset=0 "Current offset";
     parameter SI.Time startTime=0 "Time offset";
-    replaceable Modelica.Blocks.Interfaces.SignalSource signalSource(
-        final offset = offset, final startTime=startTime) annotation (Placement(
-          transformation(extent={{70,69},{91,89}})));
```
Comment (HansOlsson) [inline Modelica/Electrical/Analog/Interfaces.mo]: That change seems really good, but unrelated to the PR.
How did the signalSource appear?

File: Modelica/Magnetic/FluxTubes/Basic/Crossing.mo [hunk 8]
```diff
@@ -14,11 +14,6 @@ model Crossing "Crossing of two branches"
     "Negative port_n2 connected with port_n1"
     annotation (Placement(transformation(extent={{90,90},{110,110}})));
 
-equation
-  connect(port_p1, port_p2) annotation (Line(
-      points={{-100,100},{-100,20},{0,20},{0,-20},{100,-20},{100,-100}}, color={255,128,0}));
-  connect(port_n1, port_n2) annotation (Line(
-      points={{-100,-100},{-100,0},{100,0},{100,100}}, color={255,128,0}));
```
Comment (HansOlsson) [inline Modelica/Magnetic/FluxTubes/Basic/Crossing.mo]: Ehmm? Removing the connections from the Crossing models does not make sense!

File: Modelica/Magnetic/FluxTubes/Basic/Short.mo [hunk 9]
```diff
@@ -1,9 +1,6 @@
 within Modelica.Magnetic.FluxTubes.Basic;
 model Short "Short cut branch"
   extends Interfaces.TwoPortsElementary;
-equation
-  connect(port_p, port_n) annotation (Line(points={{-100,0},{-1,0},{-1,0},{
-          100,0}}, color={255,128,0}));
```
Comment (HansOlsson) [inline Modelica/Magnetic/FluxTubes/Basic/Short.mo]: This seems to remove the short from the Short-model. It both seems unrelated and wrong.

File: Modelica/Magnetic/FundamentalWave/Components/Crossing.mo [hunk 10]
```diff
@@ -14,11 +14,6 @@ model Crossing "Crossing of two branches"
     "Negative port_n2 connected with port_n1"
     annotation (Placement(transformation(extent={{90,90},{110,110}})));
 
-equation
-  connect(port_p1, port_p2) annotation (Line(
-      points={{-100,100},{-100,20},{0,20},{0,-20},{100,-20},{100,-100}}, color={255,128,0}));
-  connect(port_n1, port_n2) annotation (Line(
-      points={{-100,-100},{-100,0},{100,0},{100,100}}, color={255,128,0}));
```
Comment (HansOlsson) [inline Modelica/Magnetic/FundamentalWave/Components/Crossing.mo]: See previous comment on Crossing.

File: Modelica/Magnetic/FundamentalWave/Components/Short.mo [hunk 11]
```diff
@@ -1,9 +1,6 @@
 within Modelica.Magnetic.FundamentalWave.Components;
 model Short "Short cut branch"
   extends Magnetic.FundamentalWave.Interfaces.TwoPortElementary;
-equation
-  connect(port_p, port_n) annotation (Line(points={{-100,0},{-1,0},{-1,0},{
-          100,0}}, color={255,128,0}));
```
Comment (HansOlsson) [inline Modelica/Magnetic/FundamentalWave/Components/Short.mo]: See previous comment on short.

File: Modelica/Magnetic/QuasiStatic/FundamentalWave/Components/Crossing.mo [hunk 12]
```diff
@@ -9,11 +9,6 @@ model Crossing "Crossing of connections"
     annotation (Placement(transformation(extent={{90,-110},{110,-90}})));
   Interfaces.NegativeMagneticPort port_n2 "Connected with port_n1"
     annotation (Placement(transformation(extent={{90,90},{110,110}})));
-equation
-  connect(port_p1, port_p2) annotation (Line(
-      points={{-100,100},{-100,20},{0,20},{0,-20},{100,-20},{100,-100}}, color={255,170,85}));
-  connect(port_n2, port_n1) annotation (Line(
-      points={{100,100},{100,0},{-100,0},{-100,-100}}, color={255,170,85}));
```
Comment (HansOlsson) [inline Modelica/Magnetic/QuasiStatic/FundamentalWave/Components/Crossing.mo]: See previous comment on crossing.

File: Modelica/Magnetic/QuasiStatic/FundamentalWave/Components/Short.mo [hunk 13]
```diff
@@ -1,9 +1,6 @@
 within Modelica.Magnetic.QuasiStatic.FundamentalWave.Components;
 model Short "Short connection"
   extends Interfaces.TwoPortElementary;
-equation
-  connect(port_p, port_n) annotation (Line(points={{-100,0},{-1,0},{-1,0},{
-          100,0}}, color={255,170,85}));
```
Comment (HansOlsson) [inline Modelica/Magnetic/QuasiStatic/FundamentalWave/Components/Short.mo]: See previous comment on short.

File: Modelica/Thermal/HeatTransfer/Rankine/TemperatureSensor.mo [hunk 14]
```diff
@@ -3,41 +3,10 @@ model TemperatureSensor "Absolute temperature sensor in degRankine"
 
   Modelica.Blocks.Interfaces.RealOutput T annotation (Placement(
         transformation(extent={{90,-10},{110,10}})));
-  Interfaces.HeatPort_a port annotation (Placement(transformation(extent={{
-            -110,-10},{-90,10}})));
```
Comment (HansOlsson) [inline Modelica/Thermal/HeatTransfer/Rankine/TemperatureSensor.mo]: Removing the temperature sensor?

File: Modelica/ComplexBlocks.mo (hunk 4 above)
Comment (christiankral) [reply Modelica/ComplexBlocks.mo]: All these connectors are now moved on the diagram layer (only): 

image


They were not aligned correctly before:

image

File: Modelica/Electrical/Analog/Interfaces.mo (hunk 5 above)
Comment (christiankral) [reply Modelica/Electrical/Analog/Interfaces.mo]: Yes, this happened accidentally

File: Modelica/Electrical/Analog/Interfaces.mo (hunk 7 above)
Comment (christiankral) [reply Modelica/Electrical/Analog/Interfaces.mo]: Re-introuced by c4d6d1bbc65306e69062a02bf4d485e1018e6a5e

File: Modelica/Magnetic/FluxTubes/Basic/Crossing.mo (hunk 8 above)
Comment (christiankral) [reply Modelica/Magnetic/FluxTubes/Basic/Crossing.mo]: Re-introduced by fb67ce45c827f4abf485436ff2e2a9c87028801d

File: Modelica/Magnetic/FluxTubes/Basic/Short.mo (hunk 9 above)
Comment (christiankral) [reply Modelica/Magnetic/FluxTubes/Basic/Short.mo]: Re-introduced by dcb71bc74fd7f67a48d8dde6b32cdc49af9ed1e9

File: Modelica/Magnetic/FundamentalWave/Components/Crossing.mo (hunk 10 above)
Comment (christiankral) [reply Modelica/Magnetic/FundamentalWave/Components/Crossing.mo]: Re-introduced connections in e3f7cebbb0a823b28f065ccaf182fa09308297e0

File: Modelica/Magnetic/FundamentalWave/Components/Short.mo (hunk 11 above)
Comment (christiankral) [reply Modelica/Magnetic/FundamentalWave/Components/Short.mo]: Re-introduced in b39aaee0b04493cde0154b184e5c6242d2aa56c3

File: Modelica/Magnetic/QuasiStatic/FundamentalWave/Components/Crossing.mo (hunk 12 above)
Comment (christiankral) [reply Modelica/Magnetic/QuasiStatic/FundamentalWave/Components/Crossing.mo]: Re-introduce connections in 217f0dae87abb7a6bdac938aa142a4185f348d50

File: Modelica/Magnetic/QuasiStatic/FundamentalWave/Components/Short.mo (hunk 13 above)
Comment (christiankral) [reply Modelica/Magnetic/QuasiStatic/FundamentalWave/Components/Short.mo]: Re-introduced connection in 24a0c38fddd9acb50aaa994f21dc5475546834f3

File: Modelica/Thermal/HeatTransfer/Rankine/TemperatureSensor.mo (hunk 14 above)
Comment (christiankral) [reply Modelica/Thermal/HeatTransfer/Rankine/TemperatureSensor.mo]: Re-introduce heat port in Rankine.TemperatureSensor in 92975e5e94e47497f4e0fd4a5515b5ed92c043b1

**Comment by christiankral** [comment]
> ... PID has shrunk the P I and D-blocks...

Re-introduced by 58a17a1ace14ea227da9b7b00f3cfeb5d05490b2

**Comment by christiankral** [comment]
So hopefully all the deleted components are re-introduced. The affected packages are all checked OK.

File: Modelica/ComplexBlocks.mo (hunk 4 above)
Comment (christiankral) [reply Modelica/ComplexBlocks.mo]: I consider this change (somehow) relevant to this PR, as it cleans up the diagram layer.
